### PR TITLE
iPhone 15 Pro (Max) supports AV1

### PR DESCRIPTION
**Pull request by tats-u**
https://webkit.org/blog/14445/webkit-features-in-safari-17-0/

> AV1
>
> Safari 17.0 adds support for AV1 video on devices with hardware decoding support, like iPhone 15 Pro and iPhone 15 Pro Max.

I don't know whether iOS Safari supports AV1 in MP4 container too.

https://www.reddit.com/r/AV1/comments/141oqx1/new_apple_m2_promax_chips_dont_support_av1_decode/

M2 chip (Desktop/Notebook/iPad Pro) doesn't have AV1 support, so macOS shouldn't be "partially support" as of now.

The CPU in final Intel Mac is 8th gen. Intel CPU started AV1 hardware decoding in 11th gen.

---

Request to Viewers:

If you have iPhone 15 Pro (Max), I'd appreciate if you access to <https://tests.caniuse.com/av1> by your iPhone, try playing video there, and report whether you could play it or not.

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Thank you for your research and the call to action in the end, but currently arguably duplicate of https://github.com/Fyrd/caniuse/pull/6836 0:-)

**Comment by tats-u** [comment]
@Schweinepriester I know. I'll close this if you adopt my change (footnote) in your PR and add `Co-authored-by Tatsunori Uchino <tats.u@live.jp>`.

**Comment by tats-u** [comment]
We must avoid misleading iPhone 15 (Plus) or Mac or iPad with the M1/M2 chip users that they can watch AV1 videos first.

**Comment by tats-u** [comment]
@Schweinepriester I have to tell you I noticed your PR after submitting this.

**Comment by Fyrd** [comment]
@tats-u Just saw this PR after merging @Schweinepriester 's one, happy to include your updated note if you don't mind updating your PR.

**Comment by tats-u** [comment]
@Fyrd I'm willing to rebase my PR

File: features-json/av1.json [hunk 1]
```diff
@@ -584,7 +584,7 @@
     "1":"Can be enabled in Edge on Windows 10 or later by installing the [AV1 Video Extension from the Microsoft Store](https://www.microsoft.com/store/productId/9MVZQVXJBQ9V)",
     "2":"Firefox 65 only supported AV1 on Windows 64-bit",
     "3":"Firefox 66 only supported AV1 on Windows and macOS",
-    "4":"Partial support refers to supporting only devices with hardware decoding support"
+    "4":"Supported only for devices with hardware decoder i.e. A17 Pro (in e.g. iPhone 15 _Pro_ (Max)) or later"
```
Comment (tats-u) [inline features-json/av1.json:587]: @Fyrd Does this expression have to be improved?

**Comment by jensimmons** [comment]
What does this mean?
`_Pro_ (Max))`

Why the underscores and parentheses?

**Comment by tats-u** [comment]
`_Pro_` means _Pro_, and "Pro (Max)" means Pro or Pro Max

File: features-json/av1.json (hunk 1 above)
Comment (Fyrd) [reply features-json/av1.json:587]: Nah it's good, thanks!

**Review by Fyrd — APPROVED** [review]
Thanks!

**Pull request merged**